### ISSUE TITLE
Help 'transparent-background' is not recognized ！！please

### ISSUE THREAD

**Issue by wubianluomu115** [s9roll7/ebsynth_utility#71]
报错是
'transparent-background' is not recognized as an internal or external command,
operable program or batch file.
mask created

help me 。thanks
![1686204758079](https://github.com/s9roll7/ebsynth_utility/assets/127119873/65b7a27a-ca92-4bbf-a8e8-c77c71c7bccd)

**Comment by wubianluomu115** [comment]
I solved it myself. haha

**Comment by 23fbhtg34** [comment]
> 

tell me how i have the same problem

**Comment by 23fbhtg34** [comment]
I also decided to change the name of the patch latest.pth  to ckpt_base.pth

**Comment by wubianluomu115** [comment]
Let me talk about the solution：
  1.First, you need to install cmd in your D:  Stable Diffusion  scripts https://pypi.org/project/transparent-background/ Copy pip install transparent background.
  2.On your computer C:  Users  46480  AppData  Local  Packages  Python SoftwareFoundation.Python. 3.11_ Qbz5n2kfra8p0  LocalCache  local packages  Python 311  Scripts is also cmd, install pip install transparent background according to 1
  3.win+R input “sysdm.cpl"Advanced - Environment Variables PATH  Both the upper and lower levels need to be newly added, as shown in steps 1-2 
  4.download：drive.google.com/file/d/13oBl5MTVcWER3YU4fSxW3ATlVfueFQPY/view?usp=share_link     Rename ckpt_base.pth,Put in C:\Users\Account name，new-built   .transparent-background 。
  drive.google.com/file/d/1iRX-0MVbUjvAVns5MtVdng6CQlGOIo3m/view?usp=share_link   Rename ckpt_fast.pth  same
  5.Video names cannot have space symbols in Chinese。
  6.restart computer
  7.ok
 
![1686271734492](https://github.com/s9roll7/ebsynth_utility/assets/127119873/cc9acbfa-8c4a-457b-9873-a8847e178b36)
![1686271766959](https://github.com/s9roll7/ebsynth_utility/assets/127119873/8bb4a2d6-a3b4-496f-ab8f-5e0f9322282c)
![1686271807909](https://github.com/s9roll7/ebsynth_utility/assets/127119873/cd80a40e-a839-4766-97de-a911484c73bf)
![1686271838314](https://github.com/s9roll7/ebsynth_utility/assets/127119873/e8c4b1b0-9494-4838-bba7-938de9b29ea2)

**Comment by trxsudo** [comment]
> Let me talk about the solution： 1.First, you need to install cmd in your D: Stable Diffusion scripts https://pypi.org/project/transparent-background/ Copy pip install transparent background. 2.On your computer C: Users 46480 AppData Local Packages Python SoftwareFoundation.Python. 3.11_ Qbz5n2kfra8p0 LocalCache local packages Python 311 Scripts is also cmd, install pip install transparent background according to 1 3.win+R input “sysdm.cpl"Advanced - Environment Variables PATH Both the upper and lower levels need to be newly added, as shown in steps 1-2 4.download：drive.google.com/file/d/13oBl5MTVcWER3YU4fSxW3ATlVfueFQPY/view?usp=share_link Rename ckpt_base.pth,Put in C:\Users\Account name，new-built .transparent-background 。 drive.google.com/file/d/1iRX-0MVbUjvAVns5MtVdng6CQlGOIo3m/view?usp=share_link Rename ckpt_fast.pth same 5.Video names cannot have space symbols in Chinese。 6.restart computer 7.ok
> 
> ![1686271734492](https://user-images.githubusercontent.com/127119873/244555743-cc9acbfa-8c4a-457b-9873-a8847e178b36.jpg) ![1686271766959](https://user-images.githubusercontent.com/127119873/244555794-8bb4a2d6-a3b4-496f-ab8f-5e0f9322282c.jpg) ![1686271807909](https://user-images.githubusercontent.com/127119873/244555831-cd80a40e-a839-4766-97de-a911484c73bf.jpg) ![1686271838314](https://user-images.githubusercontent.com/127119873/244555880-e8c4b1b0-9494-4838-bba7-938de9b29ea2.jpg)

请教如果用秋叶整合包的如何做以上的相应修改呢？

**Comment by wubianluomu115** [comment]
same

**Comment by trxsudo** [comment]
> same

could you please make the steps more clear. it is really hard to follow. for example which windows were you using? 11 or 10? and clearly the 1 and 2 steps in your instruction has different path than mine (im using windows11)

**Comment by gj311241** [comment]
> Let me talk about the solution： 1.First, you need to install cmd in your D: Stable Diffusion scripts https://pypi.org/project/transparent-background/ Copy pip install transparent background. 2.On your computer C: Users 46480 AppData Local Packages Python SoftwareFoundation.Python. 3.11_ Qbz5n2kfra8p0 LocalCache local packages Python 311 Scripts is also cmd, install pip install transparent background according to 1 3.win+R input “sysdm.cpl"Advanced - Environment Variables PATH Both the upper and lower levels need to be newly added, as shown in steps 1-2 4.download：drive.google.com/file/d/13oBl5MTVcWER3YU4fSxW3ATlVfueFQPY/view?usp=share_link Rename ckpt_base.pth,Put in C:\Users\Account name，new-built .transparent-background 。 drive.google.com/file/d/1iRX-0MVbUjvAVns5MtVdng6CQlGOIo3m/view?usp=share_link Rename ckpt_fast.pth same 5.Video names cannot have space symbols in Chinese。 6.restart computer 7.ok
> 
> ![1686271734492](https://user-images.githubusercontent.com/127119873/244555743-cc9acbfa-8c4a-457b-9873-a8847e178b36.jpg) ![1686271766959](https://user-images.githubusercontent.com/127119873/244555794-8bb4a2d6-a3b4-496f-ab8f-5e0f9322282c.jpg) ![1686271807909](https://user-images.githubusercontent.com/127119873/244555831-cd80a40e-a839-4766-97de-a911484c73bf.jpg) ![1686271838314](https://user-images.githubusercontent.com/127119873/244555880-e8c4b1b0-9494-4838-bba7-938de9b29ea2.jpg)

请问秋叶整合包上怎么解决这个问题，他用的是自带的python，还是要在c盘创建一个transparent-background文件夹吗？
